index.js

```diff
@@ -20,8 +20,60 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    let firstIteration = Recipe.create(data[0])
+     firstIteration
+        .then((res)=>{
+              console.log("data was sent",res)
+    })
+        .catch(()=>{
+              console.log("failure")
+    })
+  
+  
+  let secondIteration = Recipe.insertMany(data)
+    secondIteration
+        .then((res)=>{
+              res.filter((res)=>{ console.log(res.title)})
+      
+               console.log("data was sent")
+    })
+        .catch(()=>{
+              console.log("failure")
+      
+    })
+  
+
+
+  Promise.all([firstIteration,secondIteration])
+
+  .then(()=>{
+    let thirdIteration =Recipe.findOneAndUpdate({title:"Rigatoni alla Genovese"},{$set:{duration:100}})
+  thirdIteration
+          .then((res)=>{
```

Comment:
since you are using promise.all later you don't need to check the promise resolution here as well with a .then and .catch. As this will already be handled inside the promise.all. It makes this code a little redundant.